src/VisualStudio/Core/Test/Snippets/CSharpSnippetExpansionClientTests.vb

```diff
@@ -314,7 +317,10 @@ using G=   H.I;
                     Guids.CSharpLanguageServiceId,
                     document.GetTextView(),
                     document.GetTextBuffer(),
-                    Nothing)
+                    signatureHelpControllerProvider:=Nothing,
+                    editorCommandHandlerServiceFactory:=Nothing,
+                    Nothing,
```

Comment:
nit: named param.

Reply:
➡️ The existing parameter was left in its current form. New parameters use named arguments. Leaving this one as-is for scope-limiting the remaining review comments.

Reply:
➡️ Updated three cases for passing `Nothing` for this parameter